noetic/Dockerfile

```
@@ -0,0 +1,82 @@
+FROM ubuntu:focal
+
+# basic args
+ENV ROS_DISTRO=noetic
+ARG USERNAME=user
```

lets change this to `container_user` and make it standard across the other containers as well, have created an issue for the same at https://github.com/soham2560/DockerForROS2Development/issues/5
```suggestion
ARG USERNAME=container_user
```

---

Done, refer [7289362](https://github.com/soham2560/DockerForROS2Development/pull/3/commits/728936217944cffb603b659eacb6ec8e3a0ff3d5)